src/main/java/org/mifos/connector/mojaloop/camel/ExternalApiCallRoute.java

```diff
@@ -0,0 +1,30 @@
+package org.mifos.connector.mojaloop.camel;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.stereotype.Component;
+
+import static org.mifos.connector.mojaloop.camel.config.CamelProperties.*;
+
+@Component
+public class ExternalApiCallRoute extends RouteBuilder {
+
+    @Override
+    public void configure() {
+        from("direct:external-api-call")
+                .id("external-api-call")
+                .log("######## API CALL -> Calling an external api")
+                .process(exchange -> {
+                    // remove the trailing "/" from endpoint
+                    String endpoint = exchange.getProperty(ENDPOINT, String.class);
+                    if (endpoint.startsWith("/")) { exchange.setProperty(ENDPOINT, endpoint.substring(1)); }
+                })
+                .log("Host: ${exchangeProperty." + HOST + "}")
+                .log("Endpoint: ${exchangeProperty." + ENDPOINT + "}")
+                .log("Headers: ${headers}")
+                .log("Request Body: ${body}")
```

Comment:
mark these logs to debug level when they are printing sensitive data

Reply:
I am configuring logging level for the root for SL4J.

Reply:
done